l10n_th_vat_report/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    'name': 'Thailand Localization - VAT Reports',
+    'version': '12.0.1.0.0',
+    'author': 'Ecosoft, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/l10n-thailand',
+    'license': 'AGPL-3',
+    'category': 'Accounting',
+    'depends': [
```

Comment:
Do we need to depend on,
* account (this already depends by l10n_th_vendor_tax_invoice)
* l10n_th_withholding_tax_cert
* l10n_th_partner
Please verify.